app.js

```diff
@@ -1,16 +1,14 @@
 const express = require('express')
 const { getAllTopics } = require('./controllers/topics.controller')
-
+const endpointsData = require('./endpoints.json')
 
 const app = express()
 
 app.get('/api/topics', getAllTopics)
 
-
-app.use((err, req, res, next) => {
-    console.log(err);
-    res.status(500).send('Server Error!');
-  });
+app.get('/api', (req, res) => {
+  res.status(200).send(endpointsData)
+})
```

Comment:
You should be using your app for your endpoints but please use the controller to res.send the endpoints 🙏